src/utilities/convert.ts

```diff
@@ -0,0 +1,15 @@
+export function convertBase64ToUnit8Array(input: string) {
+	return Buffer.from(input, "base64");
+}
+
+export function formatJSON(input: any) {
+	return JSON.stringify(input, null, 4);
+}
+
+export function convertObjectValuesToUnit8Array(input: any) {
```

Comment:
input data should be string or buffer I think

Reply:
it would be an object since we are converting it object values into Unit8Array, but we are not sure about the type of object.